app/main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+import os
+
+
+class Check:
+    forbidden_char = {"/", "\\", ":", "*", "?", '"', "<", ">", "|"}
+
+    @classmethod
+    def is_good_file(cls, file_name: str) -> bool:
+        data = os.path.splitext(file_name)
+        if not data[0] or data[1] != ".txt":
+            return False
+        return True
+
+    @classmethod
+    def is_good_name(cls, name: str) -> bool:
+        for _chr in name:
+            if _chr in cls.forbidden_char:
+                return False
+        return True
+
+    @classmethod
+    def check_path(cls, folders: list[str], file_current: str) -> bool:
+        for folder in folders:
+            if not folder or not cls.is_good_name(folder):
+                return False
+        if (
+                not cls.is_good_name(file_current)
+                or not cls.is_good_file(file_current)
+        ):
+            return False
+        return True
+
+
+def create_path(folders: list[str]) -> None:
+    path = ""
+    for folder in folders:
+        path = folder if not path else path + "/" + folder
```

Comment:
Consider using `os.path.join` instead of string concatenation for building paths. This ensures compatibility across different operating systems.